### PR TITLE
BVHLoader & example 

### PR DESCRIPTION
**Pull request by herzig**
Loader for BVH files including a simple example.

Files with multiple ROOTs (i.e. multiple characters) are not supported, I haven't come across such files yet..

### CODE REVIEW

**Comment by WestLangley** [comment]
http://threejsworker.com/api/pullrequests/9188/examples/webgl_loader_bvh.html

Thanks! You will need to include the example in `examples/files`.

**Comment by herzig** [comment]
@WestLangley Do you mean the bvh file? or the whole example? 
Also, do I have to do something so that the example shows up in the example browser (like register it somewhere)?

File: examples/webgl_loader_bvh.html [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+			var clock = new THREE.Clock();
+
+			var mixer;
+			var skeletonHelper;
+			var camera, controls, scene, renderer;
+
+			var mesh;
+
+			var loader = new THREE.BVHLoader();
+			loader.load( "models/bvh/pirouette.bvh", ( result ) => {
```
Comment (WestLangley) [inline examples/webgl_loader_bvh.html]: loader.load( "models/bvh/pirouette.bvh", ( result ) {

**Comment by WestLangley** [comment]
Add your new example to https://github.com/mrdoob/three.js/blob/dev/examples/files.js.

Your example is throwing errors -- perhaps because your mesh geometry has no faces. There likely is another way of handling this.

**Comment by mrdoob** [comment]
> http://threejsworker.com/api/pullrequests/9188/examples/webgl_loader_bvh.html

The skeleton animates but stays in the same place. Is that correct?

**Comment by herzig** [comment]
Yes, this example bvh file does not contain any position channels. I

**Comment by mrdoob** [comment]
I think so yes, people may think it's broken otherwise...

**Comment by herzig** [comment]
You're right, I agree! Also added an attribution to the original source of the animation.

**Comment by mrdoob** [comment]
Sweet! Thanks!

**Pull request merged**